server/plugin/subscriptions.go

```diff
@@ -146,22 +148,31 @@ func (p *Plugin) Subscribe(ctx context.Context, githubClient *github.Client, use
 		Repository: fullNameFromOwnerAndRepo(owner, repo),
 		Flags:      flags,
 	}
-
 	if err := p.AddSubscription(fullNameFromOwnerAndRepo(owner, repo), sub); err != nil {
 		return errors.Wrap(err, "could not add subscription")
 	}
-
 	return nil
 }
 
 func (p *Plugin) SubscribeOrg(ctx context.Context, githubClient *github.Client, userID, org, channelID, features string, flags SubscriptionFlags) error {
 	if org == "" {
 		return errors.New("invalid organization")
 	}
-
 	return p.Subscribe(ctx, githubClient, userID, org, "", channelID, features, flags)
 }
 
+func (p *Plugin) CheckIsNotificationOff(repoName string) (bool, error) {
+	subscription, err := p.GetNotificationTurnedOffRepo()
```

Comment:
I believe `subscription` is actually a list of repos.
```suggestion
	repos, err := p.GetNotificationTurnedOffRepo()
```

Reply:
done